src/openzaak/notifications/views.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: EUPL-1.2
```

Comment:
I think this file can be removed if the view from commonground-api-common is reused